code/headers/dummy_controller_c.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#include <controller_interface_c.hpp>
+
+namespace R2D2::manual_control {
+    
+    
+
+    class dummy_controller_c: public controller_interface_c{
+    private:
+    bool buttons[12] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
These look like magic numbers, I get that this is a dummy however some explanation about this class and the choices for the numbers even if there are just random would fix a lot of issues regarding the magic/random numbers.